### DISCUSSION TITLE
Support for reverse proxy base_url

### DISCUSSION THREAD

**Discussion by deepseth** [louislam/dockge#468] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

When running behind a reverse proxy, sometimes it's prefereable to use a consistent `domain.tld/<container-name>` path redirect, instead of `<container-name>.domain.tld` (e.g. when using http basic auth in front of domain.tld).

### ✔️ Solution

Enable support for a different base instead of /, and configure this via an environment variable that is passed at container creation time.

### ❓ Alternatives

Configuration could be via a file, or via a runtime paramter too.

### 📝 Additional Context

Right now, all assets (e.g. javascript files required for the front end to work) are pulled specifically from `/assets`. With the ability to specify a different base URL (e.g. `BASE_URL=/dockge`), this would rewrite requests to pull from `$BASE_URL/assets` instead (i.e. `/dockge/assets`)

**Comment by MysaaJava** [comment]
I would really need this feature.
I will try to make it on my own.
I plan on making a env variable (something like FRONTEND_BASENAME) that would be optional (defaults to '/') that one could add to the compose file.
This cannot be changed in the frontend (because we need to access the frontend in the first place ^^)
I will see if it can be done with vite env directly (i hope so)

**Reply by res80** [reply]
This would be great! Did you succeed?

**Comment by o850cHQk** [comment]
Would love this to be added in, I'm in a bit of a subdomain hell at the moment with a lot of services that need it, being able to put this into a subfolder with a reverse proxy would be great.

**Comment by non-bin** [comment]
Any work on this? it seems everything wants to be on the root of the domain, and I don't want any of it to be :P
